internal/kafka/internal/services/kafkatlscertmgmt/kafka_tls_certificate_management_service.go

```diff
@@ -147,9 +152,18 @@ func (certManagementService *kafkaTLSCertificateManagementService) RevokeCertifi
 		return nil // the certificate is revoked manually in manual mode
 	}
 
+	wildcardDomain := fmt.Sprintf("*.%s", domain)
+	refs := certManagementService.certManagementClient.GetCerticateRefs(wildcardDomain)
+
+	// No need to revoke the certificate if it does not exists
```

Comment:
It was something that was lacking but was made evident by this change. 
It could typically happen when a preparing kafka is deleted before the certificate are there.